src/features/game/types/withdrawables.ts

```diff
@@ -1436,25 +1435,20 @@ export const BUMPKIN_WITHDRAWABLES: Record<
   "Striped Yellow Shirt": () => true,
 
   // Clash of Factions Auction
-  "Non La Hat": (state) =>
-    canWithdrawTimebasedItem(new Date("2024-06-30")) &&
-    canWithdrawBoostedWearable("Non La Hat", state), // Last Auction 2024/06/29
+  "Non La Hat": (state) => canWithdrawBoostedWearable("Non La Hat", state), // Last Auction 2024/06/29
   "Oil Can": (state) =>
     canWithdrawTimebasedItem(new Date("2024-07-18")) &&
     canWithdrawBoostedWearable("Oil Can", state), // Last Auction 2024/07/17
-  "Olive Shield": (state) =>
-    canWithdrawTimebasedItem(new Date("2024-07-06")) &&
-    canWithdrawBoostedWearable("Olive Shield", state), // Last Auction 2024/07/05
+  "Olive Shield": (state) => canWithdrawBoostedWearable("Olive Shield", state), // Last Auction 2024/07/05
   "Paw Shield": () => canWithdrawTimebasedItem(new Date("2024-07-24")), // Last Auction 2024/07/23
   Pan: () => canWithdrawTimebasedItem(new Date("2024-07-21")), // Last Auction 2024/07/20
   // Clash of Factions Megastore
   "Royal Robe": () => true,
   Crown: () => true,
   "Soybean Onesie": () => canWithdrawTimebasedItem(new Date("2024-08-01")),
   "Tofu Mask": (state) =>
-    canWithdrawTimebasedItem(new Date("2024-07-01")) &&
-    canWithdrawBoostedWearable("Tofu Mask", state),
-  "Olive Royalty Shirt": () => canWithdrawTimebasedItem(new Date("2024-07-01")),
+    true && canWithdrawBoostedWearable("Tofu Mask", state),
```

Comment:
The `true` here can be removed and you can just use the return value from the `canWithdrawBoostedWearable("Tofu Mask", state)` fn